schema/descriptor.schema.json

```diff
@@ -324,16 +327,16 @@
                     }
                 }, {
                     "properties": {
-                        "type": { "enum": ["String", "File", "Number", "Enum" ] }
+                        "type": { "enum": ["String", "File", "Number"] }
                     }
                 }],
                 "dependencies": {
                     "command-line-flag-separator": ["command-line-flag"],
                     "min-list-entries": ["list"],
                     "max-list-entries": ["list"],
-                    "enum-value-choices": {
+                    "value-choices": {
                         "properties": {
-                            "type": {"enum": ["Enum"]}
+                            "type": {"enum": ["String", "Number"]}
```

Comment:
Not sure how we could specify a list of values for file... Should it be a list of URLs? Anyway, let's leave it aside for now.